### ISSUE TITLE
Class "Doctrine\DBAL\Driver\AbstractMySQLDriver" not found

### ISSUE THREAD

**Issue by timoransky** [BRACKETS-by-TRIAD/craftable-pro-docs#37]
Posted by @marzsman on Discord:

Any idea why I get this error when running the crud generator?
See screenshot

I tried to install dbal composer require doctrine/dbal but this gave me other errors like:
 ```
Doctrine\DBAL\Exception 
Unknown database type enum requested, Doctrine\DBAL\Platforms\MySQL80Platform may not support it.
```

My setup is the latest sail installation:
laravel v10.10
php v8.2.8
mysql 8.0.32 

![Screenshot_2023-07-14_at_16 29 57](https://github.com/BRACKETS-by-TRIAD/craftable-pro-docs/assets/15653065/89f33eea-5ad8-492b-9b69-23851febed3e)

**Comment by palypster** [comment]
Can we ask for the migration file as well?

**Comment by marzsman** [comment]
The migration was pretty basic.
This is the one for the categories when I had the error:

```

return new class extends Migration
{
    /**
     * Run the migrations.
     */
    public function up(): void
    {
        Schema::create('categories', function (Blueprint $table) {
            $table->id();
            $table->string('name');
            $table->string('slug')->unique();
            $table->boolean('is_active')->default(true);
            $table->timestamps();
        });
    }

    /**
     * Reverse the migrations.
     */
    public function down(): void
    {
        Schema::dropIfExists('categories');
    }
};
```

**Comment by timoransky** [comment]
Hi @marzsman 
We are looking into this issue, but we can't reproduce the part with `Unknown database type enum requested` error message since the migration you provided doesn't have any enums. Was it another migration, or did you change the column type in the meantime? 
This would help us to distinguish the two error messages from each other.
Thank you!

**Comment by marzsman** [comment]
Ah, sorry about the confusion.
I had 2 migrations, the Category migration I shared and another one.

After running the migrations without any issues, I started with the CRUD generator.

First, I had an error on the AbstractMysqlDriver. 
After looking online for a solution, I found a fix to install dbal.
That fix seemed to work for the the Category CRUD.

However, when I was running the generator for my other table, which had an enum as a datatype, I got the other error from dbal, about the enum not supported.

To continue the project, I did a rollback, changed the enum to a string and did the migration again, without any issues.

**Comment by timoransky** [comment]
Hi @marzsman 

the issue regarding missing `doctrine/dbal` composer package was fixed in the latest release [v1.1.15](https://docs.craftable.pro/changelog#v1115---2023-07-24). However, we are still working on fixing the second issue mentioning `Unknown database type enum requested` since the enum is currently not one of the database column types we support. I will let you know about any progress.

**Comment by marzsman** [comment]
Thx for the update, I'll test it asap.
At the moment, the enum fix is not an urgent issue for me because I am currently not using it anymore.
